services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from db.models import MovieSession
+from db.models import Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None

```

Comment:
session_date can be Optional. Improve type hint, please

Reply:
not fixed

use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

Reply:
@AnatoliyPilipey it is not fixed